packages/typespec-vscode/src/tsp-language-client.ts

```diff
@@ -108,7 +108,7 @@ export class TspLanguageClient {
       return result;
     } catch (e) {
       logger.error(`Unexpected error when running Cli command with args [${args.join(", ")}]`, [e]);
-      return undefined;
+      throw e;
```

Comment:
any reason to change to throw which would require all the caller to try-catch it properly. An unhandled exception will break the whole extension. Returning undefined ha already indicated the function failed for some reason, isn't that enough?

Reply:
rolled back to return undefined